CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## [1.9.7-5] - 2019-07-12
+### Fix
+- Fix 'destroy' application method - Add checking to not fail in case component doesn't exist
+- Fix busbar Docker build - fix apt repository string
+
```

Comment:
Wondering your thoughts on the following?:

```suggestion
## [1.9.7-5] - 2019-07-12
### Fixed
- 'destroy' application method (add checking to not fail in case component doesn't exist)
- Busbar Docker build by updating apt repository string
```